### PR TITLE
[PLATSUP-1016] ZT FAQ PP Blurbs

### PR DESCRIPTION
**Pull request by cameronjp**
https://zapierorg.atlassian.net/browse/PLATSUP-1016

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1707867994316959) and give us a `:review-pr:` emoji.

**Comment by Osarhomiyeke** [comment]
Hi @cameronjp on the FAQ section of the Zap Templates doc, was it your intention for the answers to be on the same line as the questions ([screenshot](https://cdn.zappy.app/43a16cd4ef785998407b40a95dbbf8f2.png)) or did you intend for the answers to always be on a new line?

File: docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Your integration is a powerful tool for automating workflow, increasing productivity, and providing value to your users. Including it in your product expansion strategy and product roadmap planning, ensures it evolves in tandem with your core offerings.
+
+> Pro tip: Read on how you can [prioritize user feedback](https://zapier.com/blog/prioritize-user-feedback-with-beamer/) in your product roadmap planning.
+
+## 4. Spread the word about your integration's new features
+Highlight new features and enhancements to your integration in your marketing materials. Adding these to newsletters, blogs, and in-product messages (IPMs), can help drive engagement and encourage users to explore new functionalities, leading to increased adoption and deeper integration into their daily workflows.
+
+> Pro Tip: Get your integration in front of more users by embedding Zapier’s [Partner Solutions](https://zapier.com/partner/solutions/plug-and-play) directly into your product. 
+
+## 5. Keep Your Integration Fresh with Regular Updates
```
Comment (marinahand) [inline docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md:29]: Keep your integration fresh with regular updates
(sentence case to follow developer style guide from Tech Writing)

File: docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Regular updates signal to your users that you are committed to continuously improving their experience. Establish a cadence for updates that align with your sprint cycles.
+
+> Pro tip: Add them to your calendar.
+
+## Next Steps:
```
Comment (marinahand) [inline docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md:34]: Next Steps
(remove : as shows in the side bar headings)

File: docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- Actively monitor user feedback
+- [Leave a comment](https://platform.zapier.com/manage/user-feedback) on the Bugs & Feature Requests section inside your Partner Dashboard
+- When promoting a version, link any open bug reports and feature requests that the newly promoted version addresses
+    - _Only possible when promoting within the developer platform_
```
Comment (marinahand) [inline docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md:38]: Only possible when promoting within the Platform UI
(how it is referred to in rest of docs)

File: docs/_publish/launch-process/zap-templates.md [hunk 4]
```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
 When deprecating a version of your integration, any Zap templates using the integration will automatically be marked as _Invalid_ within 24 hours of the deprecation.
 
 While invalid, the Zap template will not be _Public_ until it is adjusted to use a non-deprecated version, re-submitted for review and approved. This ensures users won’t use deprecated and broken Zap templates to make Zaps. If you’re having trouble accessing _Invalid_ templates, please [submit a ticket.](https://developer.zapier.com/contact)
+
+## Frequenly Asked Questions
+
+**Q:** How come I can see Zap templates on my directory page that don’t appear in the developer platform?
```
Comment (marinahand) [inline docs/_publish/launch-process/zap-templates.md:275]: Why do I see Zap templates on my app's directory page that don’t appear under my [Zap Templates](https://developer.zapier.com/zap-templates).

File: docs/_publish/launch-process/zap-templates.md [hunk 5]
```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
...
+
+**Q:** How can I make changes to a published Zap template?
+**A:** Once a Zap template is published, you are unable to self-serve changes to it. To request changes, please submit our [contact form](https://developer.zapier.com/contact). When submitting the contact form, please ensure you provide the Zap template ID and the change you’re requesting per Zap template. The Zap template ID can be found by clicking on the [3-dot icon](https://cdn.zappy.app/227b0c1da48e72115ced60cdb8c06a97.png) in the developer platform, or [in the URL](https://cdn.zappy.app/c913c363b267fd1c4c33b5395e092460.png) when viewing a Zap template landing page. Please note the following when requesting changes:
+- Zap templates should work for the broadest audience. Try to avoid requesting changes that will exclude an audience segment from successfully using a Zap template
+- Every partner on Zapier has their own style and tone. We will not update titles or descriptions of Zap templates created by other partners* to better accommodate your preferences. While a title and description might not match your tone and style, it could very well for the partner that published the Zap template
```
Comment (marinahand) [inline docs/_publish/launch-process/zap-templates.md:284]: partners* - is that a typo with the * ?

File: docs/_publish/launch-process/zap-templates.md [hunk 6]
```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
...
+  - *If you believe you found an error with a title or description (eg the title references the wrong action), we will review and consider making the correction
```
Comment (marinahand) [inline docs/_publish/launch-process/zap-templates.md:285]: purpose of the *

File: docs/_publish/launch-process/zap-templates.md [hunk 7]
```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
...
+
+**Q:** How many Zap templates should my integration have?
+**A:** The more the merrier! One Zap template isn’t enough—you’ll want to make Zap templates for each of your app’s most popular use cases. Although there’s not a set number, we recommend adding 5 to 10 Zap templates to your integration, for a start. Learn more about Building Zap templates [here](https://platform.zapier.com/publish/zap-templates#how-to-build-a-zap-template).
+
+**Q:** Why did my Zap Templates stop working?
+**A:** Common reasons your Zap template stopped working include breaking changes, versioning, migration, and authentication. Adding, updating, replacing, or deleting components can have various effects. Learn more about Planning and implementing integration changes [here](https://platform.zapier.com/manage/planning-changes).
```
Comment (marinahand) [inline docs/_publish/launch-process/zap-templates.md:320]: planning and implementing integration changes (lower case p)

File: docs/_publish/launch-process/zap-templates.md [hunk 8]
```diff
@@ -269,3 +269,61 @@ If breaking changes exist between the previous and newly promoted integration ve
...
+
+**Q:** Can I add filters/other conditions to my Zap templates?
+**A:** Yes. Most Zap templates only need a trigger and two action steps. However, we know sometimes you may need more steps for advanced workflows. Learn more [here](https://platform.zapier.com/publish/zap-templates#3-optional-add-filters-or-additional-action-steps).
+
+**Q:** Do users need to have an account to create their own Zaps?
+**A:** All users requires their own Zapier account in order to create Zaps, even if the Zap originates from a Zap template. The account the Zap is owned by will house all of the Zap particulars, such as the authorizations used for the Zap to work. To help reduce friction for new users, you should pair your embed solution with our [Quick Account Creation](https://platform.zapier.com/embed/quick-account-creation). This seamless, accelerated sign-up feature allows first-time Zapier users to skip the standard sign-up procedure and onboarding survey.
```
Comment (marinahand) [inline docs/_publish/launch-process/zap-templates.md:329]: All users requires their own Zapier account in order to create Zaps, even if the Zap originates from a Zap template. The account the Zap is owned by will add and store authentications for each app used in the Zap. To help reduce friction for new users, you should pair your embed solution with our [Quick Account Creation](https://platform.zapier.com/embed/quick-account-creation). This seamless, accelerated sign-up feature allows first-time Zapier users to skip the standard sign-up procedure and onboarding survey.

File: docs/_publish/partner-program/benefits-guide.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  }
+
+.even-columns th, .even-columns td {
+    width: 25%;
+    text-align: left; /* or center, if you prefer */
+  }
+</style>
+
+# A Guide to Zapier Partner Program Benefits: _Accelerate your integration's growth and reach_
+Zapier offers a variety of marketing and support benefits to partners.. This cheat sheet is designed to help you understand when and how you can access each of these benefits as you unlock them.
```
Comment (marinahand) [inline docs/_publish/partner-program/benefits-guide.md:21]: double ..

File: docs/_publish/launch-process/zap-templates.md [hunk 10]
```diff
@@ -269,3 +269,76 @@ If breaking changes exist between the previous and newly promoted integration ve
 When deprecating a version of your integration, any Zap templates using the integration will automatically be marked as _Invalid_ within 24 hours of the deprecation.
 
 While invalid, the Zap template will not be _Public_ until it is adjusted to use a non-deprecated version, re-submitted for review and approved. This ensures users won’t use deprecated and broken Zap templates to make Zaps. If you’re having trouble accessing _Invalid_ templates, please [submit a ticket.](https://developer.zapier.com/contact)
+
+## Frequenly Asked Questions
```
Comment (marinahand) [inline docs/_publish/launch-process/zap-templates.md:273]: typo - Frequently

**Review by marinahand — APPROVED** [review]

**Pull request merged**